_user/Makefile

```diff
@@ -0,0 +1,28 @@
+#
+# Makefile for user app
+#
+# Copyright 2021 Phoenix Systems
+#
+# %LICENSE%
+#
+
+include ../phoenix-rtos-build/Makefile.common
+
+CFLAGS += $(BOARD_CONFIG)
+
+.DEFAULT_GOAL := all
+
+DEFAULT_COMPONENTS := hello
+
+# default path for the programs to be installed in rootfs
+DEFAULT_INSTALL_PATH := /userbin
+
+# read out all components
+ALL_MAKES := $(wildcard */Makefile) $(wildcard */*/Makefile) $(wildcard */*/*/Makefile)
```

Comment:
too many wildcards, safe to use only the first one or `find` approach from `phoenix-rtos-test`

Reply:
I agree with  Marek.